### PR TITLE
[fix](log) Support fe log rollover size strategy

### PR DESCRIPTION
**Pull request by deardeng**
## Proposed changes

Issue Number: close #xxx

Support fe log rollover size strategy, default use age strategy. cloud use size stragegy

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by deardeng** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -2606,6 +2606,11 @@ public class Config extends ConfigBase {
     //==========================================================================
     //                    begin of cloud config
     //==========================================================================
+    @ConfField(description = {"是否启用FE 日志文件按照大小删除策略，当日志大小超过指定大小，删除相关的log。默认关闭，为按照时间策略删除",
+        "Whether to enable the FE log file deletion policy based on size, "
+            + "where logs exceeding the specified size are deleted. "
+            + "It is disabled by default and follows a time-based deletion policy."})
+    public static boolean log_by_size = false;
```
Comment (morningman) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:2613]: Better use a enum type like:
`public static String log_rollover_strategy = "age"`

The options are `size` or `age`.

And don't forget to add the `option` field of `@ConfField`

File: fe/fe-core/src/main/java/org/apache/doris/common/Log4jConfig.java [hunk 2]
```diff
@@ -39,81 +39,114 @@
...
+        // CHECKSTYLE OFF
+        xmlConfTemplateBuilder.append("<?xml version=\"1.0\" encoding=\"utf-8\"?>\n")
+            .append("\n<!-- Auto Generated. DO NOT MODIFY IT! -->\n")
+            .append("<Configuration status=\"info\" packages=\"org.apache.doris.common\">\n")
+            .append("  <Appenders>\n")
+            .append("    <Console name=\"Console\" target=\"SYSTEM_OUT\">")
+            .append("      <PatternLayout charset=\"UTF-8\">\n")
+            .append("        <Pattern>%d{yyyy-MM-dd HH:mm:ss,SSS} %p (%t|%tid)<!--REPLACED BY LOG FORMAT-->%m%n</Pattern>\n")
+            .append("      </PatternLayout>\n")
+            .append("    </Console>")
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/Log4jConfig.java:54]:
```suggestion
            .append("    </Console>\n")
```

**Review by morningman — COMMENTED** [review]
And please use English to describe your PR

**Comment by deardeng** [comment]
run buildall

**Comment by deardeng** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Comment by dataroaring** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**